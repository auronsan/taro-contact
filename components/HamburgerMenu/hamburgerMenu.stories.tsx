import React from 'react';
import { HamburgerMenu } from './index';

export default {
  title: 'components/HamburgerMenu',
  component: HamburgerMenu,
};

export const Default = () => <HamburgerMenu menu={<div>Menu Content</div>} />;

export const WithOpen = () => <HamburgerMenu menu={<div>Menu Content</div>} />;
