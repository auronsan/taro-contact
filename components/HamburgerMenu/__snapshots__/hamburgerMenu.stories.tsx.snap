// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/HamburgerMenu Default smoke-test 1`] = `
<main class="roboto-normal">
  <div class="hambugerMenu_hamburger-menu__z2Kmd">
    <button type="button"
            class="button_root__s7zfG button_button-primary__cWRWo button_button-variant-filled__Dz0lC"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           width="24"
           height="24"
           viewbox="0 0 24 24"
           fill="none"
           stroke="currentColor"
           stroke-width="2"
           stroke-linecap="round"
           stroke-linejoin="round"
           class="tabler-icon tabler-icon-menu-2 "
      >
        <path d="M4 6l16 0">
        </path>
        <path d="M4 12l16 0">
        </path>
        <path d="M4 18l16 0">
        </path>
      </svg>
    </button>
    <div class="hambugerMenu_hamburger-menu-content__Sy1U0">
      <div class="group_root__JMRrN group_justify-space-between__ZZe3p group_gap-sm__VCWmB"
           style="padding: 30px;"
      >
        <span class="text_text__ynNTn text_fs-md__gO30j">
          Taro Contact
        </span>
        <button type="button"
                class="button_root__s7zfG button_button-primary__cWRWo button_button-variant-filled__Dz0lC"
        >
          <svg xmlns="http://www.w3.org/2000/svg"
               width="24"
               height="24"
               viewbox="0 0 24 24"
               fill="none"
               stroke="currentColor"
               stroke-width="2"
               stroke-linecap="round"
               stroke-linejoin="round"
               class="tabler-icon tabler-icon-x "
          >
            <path d="M18 6l-12 12">
            </path>
            <path d="M6 6l12 12">
            </path>
          </svg>
        </button>
      </div>
      <div>
        Menu Content
      </div>
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/HamburgerMenu WithOpen smoke-test 1`] = `
<main class="roboto-normal">
  <div class="hambugerMenu_hamburger-menu__z2Kmd">
    <button type="button"
            class="button_root__s7zfG button_button-primary__cWRWo button_button-variant-filled__Dz0lC"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           width="24"
           height="24"
           viewbox="0 0 24 24"
           fill="none"
           stroke="currentColor"
           stroke-width="2"
           stroke-linecap="round"
           stroke-linejoin="round"
           class="tabler-icon tabler-icon-menu-2 "
      >
        <path d="M4 6l16 0">
        </path>
        <path d="M4 12l16 0">
        </path>
        <path d="M4 18l16 0">
        </path>
      </svg>
    </button>
    <div class="hambugerMenu_hamburger-menu-content__Sy1U0">
      <div class="group_root__JMRrN group_justify-space-between__ZZe3p group_gap-sm__VCWmB"
           style="padding: 30px;"
      >
        <span class="text_text__ynNTn text_fs-md__gO30j">
          Taro Contact
        </span>
        <button type="button"
                class="button_root__s7zfG button_button-primary__cWRWo button_button-variant-filled__Dz0lC"
        >
          <svg xmlns="http://www.w3.org/2000/svg"
               width="24"
               height="24"
               viewbox="0 0 24 24"
               fill="none"
               stroke="currentColor"
               stroke-width="2"
               stroke-linecap="round"
               stroke-linejoin="round"
               class="tabler-icon tabler-icon-x "
          >
            <path d="M18 6l-12 12">
            </path>
            <path d="M6 6l12 12">
            </path>
          </svg>
        </button>
      </div>
      <div>
        Menu Content
      </div>
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
